stl/inc/flat_set

```diff
@@ -506,8 +561,8 @@ private:
     template <class _Other>
         requires _Keylt_transparent || is_same_v<_Other, _Kty>
     _NODISCARD const_iterator _Find(const _Other& _Val) const {
-        const iterator _End   = end();
-        const iterator _Where = lower_bound(_Val);
+        const const_iterator _End   = cend();
+        const const_iterator _Where = lower_bound(_Val);
         if (_Where != _End && _Keys_equal(*_Where, _Val)) {

```

Comment:
As a side note, most of usages of `_Keys_equal` can be simplified to a single comparision, as it is used after `lower_bound`. However I think it's better not to include such changes in this pr.